### ISSUE TITLE
Daemon does not work with rails_config

### ISSUE THREAD

**Issue by teamon** [Locale/localeapp#92]
`config/initializers/localeapp.rb`

``` ruby
require 'localeapp/rails'

Localeapp.configure do |config|
  config.api_key = Settings.my_app.localeapp_api_key
  config.polling_environments = [:development]
end
```

```
$ localeapp daemon
error: uninitialized constant Settings
```

where `Settings` came from 

`config/initializers/rails_config.rb`

``` ruby
RailsConfig.setup do |config|
  config.const_name = "Settings"
end
```

As I see, localeapp is looking explicitly for `config/localeapp.rb` file and then loads it without loading rails environment - this way it is pretty pointless making it an ruby initializer as it leads to conflict like above, and one can't use any other rails or 3rd party library code in that initializer.

**Comment by redbar0n** [comment]
I just hit this problem, trying to run the daemon locally like this:
```
› localeapp daemon -b
error: uninitialized constant Rails
```
After adding this to the initializer file (to have Heroku pull the latest translations when it starts or restarts the dyno):
```
if Rails.env.production?
  Localeapp::CLI::Pull.new.execute
end
```

As was suggested by your wiki guide:
https://github.com/Locale/localeapp/wiki/Deploy-a-Locale-Backed-Application-to-Heroku

I agree with @teamon and @tushar that when localeapp is placed in the Rails initializers folder (like the documentation suggests: `config/initializers/localeapp.rb`), you expect the rails environment to be loaded. When it's not, it's misleading. Especially since all other apps allow you to reference Rails.env from within the initializer file (without requiring anything). Just see the docs for [carrierwave](https://github.com/carrierwaveuploader/carrierwave) and [delayed_job](https://github.com/collectiveidea/delayed_job) and search on those pages for "initializer", and you'll see it.

It seems that people have successfully used Rails.env from within the initializer file, since you suggest it in the wiki guide, and in [issue 60](https://github.com/Locale/localeapp/issues/60)

*How can I work around this and still use Rails.env from within the initializer file?*
When I added `require 'rails_config'` to it, I just got `cannot load such file -- rails_config` when running the daemon..
